wordpress-mobile/WordPress-iOS
# [Media Editing] Load iCloud Photos

Fixes https://github.com/wordpress-mobile/MediaEditor-iOS/issues/10

### To test

1. Try to edit any media that only exists in iCloud (and it's not stored in your device)
2. Check that the image loads correctly

PR submission checklist:
